sys/include/net/ng_pktbuf.h

```diff
@@ -57,33 +57,37 @@ extern "C" {
 #endif  /* NG_PKTBUF_SIZE */
 
 /**
+ * @brief   Initializes packet buffer module.
```

Comment:
I never spent thoughts on this before. Which module do you have in mind? Anyway, if we are talking about "comparable" situations I can live with not writing it down here.